### PR TITLE
Create an ads/google directory

### PR DESCRIPTION
**Pull request by bobcassels**
Move ads/adsense.* and ads/doubleclick.* to ads/google/
Add ads/google/utils.js, as a place for code to be shared by adsense.js and doubleclick.js.

### CODE REVIEW

**Comment by bobcassels** [comment]
I signed it!

**Comment by jridgewell** [comment]
Lol, @googlebot. @bobcassels, just ignore it.

File: ads/google/utils.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {!Object} data
```
Comment (dvoytenko) [inline ads/google/utils.js]: I realize it's been here before, but could you please update `@param` here? It should be
```
@param {!Window} global
```

**Comment by dvoytenko** [comment]
LGTM. Just one small request.

**Comment by paislee** [comment]
Won't this break anything that depends on doubleclick code? e.g. https://github.com/ampproject/amphtml/blob/master/ads/openx.js#L18

**Comment by dvoytenko** [comment]
@paislee - no, it's a simple file move. openx code is updated in this PR. But do please take a look if update as you'd expect.

**Comment by paislee** [comment]
@dvoytenko thanks, missed that. Looks good

**Comment by bobcassels** [comment]
@dvoytenko Fixed that type. Also fixed amp-ad.md.

File: ads/google/utils.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {!Window} data
```
Comment (dvoytenko) [inline ads/google/utils.js]: Almost ;) Please also rename "data" to "global" here to reflect the actual argument name below.

**Comment by dvoytenko** [comment]
@bobcassels Thanks! Just one more small comment. And please squash commits - I'll be able to merge then.

**Comment by bobcassels** [comment]
Oops. Apparently I screwed up the rebase, and grabbed a bunch of changes I didn't want. I'll figure out how to undo that.

**Comment by jridgewell** [comment]
`git pull --rebase ampproject master`, given `ampproject` is the name of the ampproject/amphtml remote in your local clone.

**Comment by bobcassels** [comment]
Rebased, and collapsed to a single commit.

**Comment by dvoytenko** [comment]
Thanks! Merging.

**Pull request merged**